### PR TITLE
API re-work; using PUT for updates instead of PATCH

### PR DESCRIPTION
**Pull request by kiambogo**

### CODE REVIEW

File: api/api.json [hunk 1]
```diff
@@ -32,9 +32,7 @@
           "description": "Road consisting of paved asphalt"
         },
         {
-          "name": "dirt",
-          "description": "Road consisting of cleared dirt"
-        }
+          "name": "dirt", "description": "Road consisting of cleared dirt" }
```
Comment (wittjosiah) [inline api/api.json]: weird spacing here

File: api/api.json [hunk 2]
```diff
@@ -147,6 +145,35 @@
         }
       ]
     },
+    "segmentUpdateForm": {
+      "description": "Form to modify an existing segment",
+      "fields": [
+        { "name": "name",
```
Comment (wittjosiah) [inline api/api.json]: also weird spacing in fields here

**Review by wittjosiah — APPROVED** [review]

File: api/api.json (hunk 2 above)
Comment (kiambogo) [reply api/api.json]: how so?

Comment (wittjosiah) [reply api/api.json]: data starts on the same line as `{`

Comment (kiambogo) [reply api/api.json]: you realize that's how everything is in this file? :)

Comment (wittjosiah) [reply api/api.json]: not everything 😛 https://github.com/trifectalabs/roadquality/pull/13/files/3349ae90fa3131b17dacae5214d73a20fba0cec2#diff-4b95db13161ad2c8cfa9f6d16d3858a6R260

**Pull request merged**